src/main/java/uk/gov/digital/ho/hocs/search/application/aws/config/LocalStackConfiguration.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.digital.ho.hocs.search.application.aws.config;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.sqs.AmazonSQSAsync;
+import com.amazonaws.services.sqs.AmazonSQSAsyncClientBuilder;
+import com.amazonaws.services.sqs.model.PurgeQueueRequest;
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.aws.messaging.config.SimpleMessageListenerContainerFactory;
+import org.springframework.cloud.aws.messaging.config.annotation.EnableSqs;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+
+@EnableSqs
+@Configuration
+@Profile("localstack")
+public class LocalStackConfiguration {
+
+    @Primary
+    @Bean
+    public AmazonSQSAsync awsSqsClient(
+            @Value("${aws.sqs.search.name}") String queueName,
+            @Value("${aws.sqs.search.url}") String awsBaseUrl,
+            @Value("${aws.sqs.search.access-key}") String accessKey,
```

Comment:
These don't need to be pulled through, can be just hard-code as "TEST". Upto you though...